Shared/Extensions/URLExtensions.swift

```diff
@@ -89,7 +89,7 @@ extension URL {
 }
 
 // The list of permanent URI schemes has been taken from http://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml
-private let permanentURISchemes = ["aaa", "aaas", "about", "acap", "acct", "cap", "cid", "coap", "coaps", "crid", "data", "dav", "dict", "dns", "example", "file", "ftp", "geo", "go", "gopher", "h323", "http", "https", "iax", "icap", "im", "imap", "info", "ipp", "ipps", "iris", "iris.beep", "iris.lwz", "iris.xpc", "iris.xpcs", "jabber", "ldap", "mailto", "mid", "msrp", "msrps", "mtqp", "mupdate", "news", "nfs", "ni", "nih", "nntp", "opaquelocktoken", "pkcs11", "pop", "pres", "reload", "rtsp", "rtsps", "rtspu", "service", "session", "shttp", "sieve", "sip", "sips", "sms", "snmp", "soap.beep", "soap.beeps", "stun", "stuns", "tag", "tel", "telnet", "tftp", "thismessage", "tip", "tn3270", "turn", "turns", "tv", "urn", "vemmi", "vnc", "ws", "wss", "xcon", "xcon-userid", "xmlrpc.beep", "xmlrpc.beeps", "xmpp", "z39.50r", "z39.50s"]
+private let permanentURISchemes = ["aaa", "aaas", "about", "acap", "acct", "cap", "cid", "coap", "coaps", "crid", "data", "dav", "dict", "dns", "dtn", "example", "file", "ftp", "geo", "go", "gopher", "h323", "http", "https", "iax", "icap", "im", "imap", "info", "ipn", "ipp", "ipps", "iris", "iris.beep", "iris.lwz", "iris.xpc", "iris.xpcs", "jabber", "ldap", "leaptofrogans", "mailto", "mid", "msrp", "msrps", "mtqp", "mupdate", "news", "nfs", "ni", "nih", "nntp", "opaquelocktoken", "pkcs11", "pop", "pres", "reload", "rtsp", "rtsps", "rtspu", "service", "session", "shttp", "sieve", "sip", "sips", "sms", "snmp", "soap.beep", "soap.beeps", "stun", "stuns", "tag", "tel", "telnet", "tftp", "thismessage", "tip", "tn3270", "turn", "turns", "tv", "urn", "vemmi", "vnc", "ws", "wss", "xcon", "xcon-userid", "xmlrpc.beep", "xmlrpc.beeps", "xmpp", "z39.50r", "z39.50s"]
```

Comment:
It may be safe to remove `shttp` from this list now, given [information here from IETF](https://datatracker.ietf.org/doc/status-change-http-experiments-to-historic/). RFC2660 is listed as Historic, but IANA states it's OBSOLETE. 

There are other URI Schemes that have the status of Permanent but are missing in this list:
- coap+tcp
- coap+ws
- coaps+tcp
- coaps+ws

Should we continue excluding these aliases? @nbhasin2 @st3fan

Reply:
There is a footnote at the bottom that says:

> CoAP registers different URI schemes for accessing CoAP resources 
> via different protocols. This approach runs counter to the WWW principle that a 
> URI identifies a resource and that multiple URIs for identifying the same 
> resource should be avoided

and it has been permanent since at least 2018.

Reply:
Yup! URI aliases aren't advised, but given the list hasn't been touched in a while (outside of the `javascript` removal), I thought it'd be worth evaluating again. 

Hopefully we get our answer soon! I'll circle back when we have something.